pages/services/kafka/2.6.0-2.2.1/reference/index.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: API reference for DC/OS Apache Kafka
+title: API Reference
+menuWeight: 90
+model: /services/kafka/data.yml
+render: mustache
+---
+
+#include /services/include/api-reference.tmpl
+
+# Topic Operations
+
+These operations mirror what is available with `bin/kafka-topics.sh`.
+
+## List Topics
+
+```bash
+dcos {{ model.packageName }} --name={{ model.serviceName }} topic list
+[
+  "topic1",
+  "topic0"
+]
+```
+
+```bash
+curl -H "Authorization: token=$auth_token" "<dcos_url>/service/{{ model.serviceName }}/v1/topics"
+[
+  "topic1",
+  "topic0"
+]
+```
+
+## Describe Topic
+
+```bash
+dcos {{ model.packageName }} --name={{ model.serviceName }} topic describe topic1
+{
+  "partitions": [
+  {
+    "0": {
+      "controller_epoch": 1,
+      "isr": [
+        0,
+        1,
+        2
+      ],
+      "leader": 0,
+      "leader_epoch": 0,
+      "version": 1
+    }
+  },
+  {
+    "1": {
+      "controller_epoch": 1,
+      "isr": [
+        1,
+        2,
+        0
+      ],
+      "leader": 1,
+      "leader_epoch": 0,
+      "version": 1
+    }
+  },
+  {
+    "2": {
+      "controller_epoch": 1,
+      "isr": [
+        2,
+        0,
+        1
+      ],
+      "leader": 2,
+      "leader_epoch": 0,
+      "version": 1
+    }
+  }
+  ]
+}
+```
+
+```bash
+curl -X POST -H "Authorization: token=$auth_token" "<dcos_url>/service/{{ model.serviceName }}/v1/topics/topic1"
+{
+  "partitions": [
+  {
+    "0": {
+      "controller_epoch": 1,
+      "isr": [
+        0,
+        1,
+        2
+      ],
+      "leader": 0,
+      "leader_epoch": 0,
+      "version": 1
+    }
+  },
+  {
+    "1": {
+      "controller_epoch": 1,
+      "isr": [
+        1,
+        2,
+        0
+      ],
+      "leader": 1,
+      "leader_epoch": 0,
+      "version": 1
+    }
+  },
+  {
+    "2": {
+      "controller_epoch": 1,
+      "isr": [
+        2,
+        0,
+        1
+      ],
+      "leader": 2,
+      "leader_epoch": 0,
+      "version": 1
+    }
+  }
+  ]
+}
+
+```
+
+## Create Topic
+
+```bash
+dcos {{ model.packageName }} --name={{ model.serviceName }} topic create topic1 --partitions=3 --replication=3
+{
+  "message": "Output: Created topic \"topic1\"\n"
+}
+```
+
+```bash
+curl -X POST -H "Authorization: token=$auth_token" "<dcos_url>/service/{{ model.serviceName }}/v1/topics/topic1?partitions=3&replication=3"
+{
+  "message": "Output: Created topic \"topic1\"\n"
+}
+```
+
+## View Topic Offsets
+
+There is an optional `--time` parameter which may be set to either "first", "last", or a timestamp in milliseconds as [described in the {{ model.techShortName }} documentation](https://kafka.apache.org/documentation/#topicconfigs).
```

Comment:
Reqrite `There is an optional `--time` parameter which may be set to either "first", "last", or a timestamp in milliseconds as [described in the {{ model.techShortName }} documentation](https://kafka.apache.org/documentation/#topicconfigs).` 

with 
`There is an optional `--time` parameter which may be set to either "first", "last", or a timestamp in milliseconds as described in the [{{ model.techShortName }} documentation](https://kafka.apache.org/documentation/#topicconfigs).